.github/workflows/update_sdk_methods.py

```diff
@@ -1046,13 +1046,37 @@ def parse(type, names):
                                     ## Split contained text at " - " to get first half, which is just the error name:
                                     raises_name = regex.split(r" – ", strong_tag.parent.text)[0]
 
-                                    ## OPTION: Get full error raised usage, including type info and html links if present.
-                                    ## NOTE: Errors raised (py) don't seem to have any links, just some monospace formatting:
-                                    ## NOTE: Currently unused.
-                                    this_method_raises_dict["raises_usage"] = str(strong_tag.parent).replace("\n", " ")
+                                    ## Process remaining data fields depending on whether the error data type is linked or not.
+                                    ## If the error includes a linked data type:
+                                    if strong_tag.parent.name == 'a':
+
+                                        ## OPTION: Get full error raised usage, including type info and html links if present.
+                                        ## NOTE: Currently unused.
+                                        raises_usage = str(strong_tag.parent.parent).replace("\n", " ")
+                                        raises_link = strong_tag.parent.get('href')
+
+                                        ## Replace the scraped relative link with a full URL, in one of two forms:
+                                        ## Scraped link is an anchor link:
+                                        if raises_link.startswith('#'):
+                                            this_method_raises_dict["raises_usage"] = regex.sub(r'href=".*"', 'href="' + url + raises_link + '"', raises_usage)
+                                        ## Scraped link is a relative link:
+                                        elif raises_link.startswith('../'):
+                                            this_method_raises_dict["raises_usage"] = = regex.sub(r'href=".*"', 'href="' + sdk_url + '/autoapi/viam/' + raises_link.replace('../', '') + '"', raises_usage)
```

Comment:
I don't this this runs currently
```suggestion
                                            this_method_raises_dict["raises_usage"] = regex.sub(r'href=".*"', 'href="' + sdk_url + '/autoapi/viam/' + raises_link.replace('../', '') + '"', raises_usage)
```

Reply:
Oops, thank you!